### ISSUE TITLE
Update scripts for Babbage 

### ISSUE THREAD

**Issue by jfischoff** [woofpool/cardano-private-testnet-setup#23]
Hi all,

First, thanks for making these scripts, they have been invaluable for my development via Canonical LLC.

I would love to see them upgraded to support Babbage/Vasil.

**Comment by extramileit** [comment]
Yes, good idea.  I personally wont have time to work on that for a few weeks, but will do some investigation when I have some bandwidth unless somebody submits a PR before.

**Comment by extramileit** [comment]
@jfischoff ,
Please have a look at the `babbage` branch.  I took the script from IOHK to start up nodes in `babbage` era and tweaked them as necessary to be compatible with latest `cardano-db-sync`.  I also updated the directions for building `cardano-db-sync` and so on. The nice thing about the newer version of `cardano-db-sync` is that you do not need to cherry pick certain commits to make db-sync compatible with private network.

I'd like to add more instructions on using the new `plutus V2` capabilities and running plutus script transactions, but don't have the time right now.

Eventually, I'd like to merge this branch into `main` branch, but want to update the additional smart contract instructions and wait for the Vasil Hard Fork to be completed, so this guide is in-line with Cardano Mainnet.

Thanks
